Sprint-3/2-practice-tdd/count.test.js

```diff
@@ -22,3 +22,44 @@ test("should count multiple occurrences of a character", () => {
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+test("should return 0 if the character doesn't appear in the string", () => {
+  const str = "aaaaa";
+  const char = "s";
+  const count = countChar(str, char);
+  expect(count).toEqual(0);
+});
+
+test("should return 0 if the character is empty", () => {
+  const str = "aaaaa";
+  const char = "";
+  const count = countChar(str, char);
+  expect(count).toEqual(0);
+});
+
+test("should return 0 if the character doesn't appear in the string", () => {
+  const str = "aaca";
+  const char = "ab";
+  const count = countChar(str, char);
+  expect(count).toEqual(0);
+});
+
+test("should return 0 if the character doesn't appear in the string", () => {
```

Comment:
This is fine if this is the behaviour you designed for your function but if `str` is empty, can there be any character? Should the function have a guardrail to protect against this corner case?

It's fine to leave this as is, you did a great job on adding the tests. It's just something to think about.